samples/word/04-range/split-words-of-first-paragraph.yaml

```diff
@@ -4,59 +4,57 @@ description: 'Show how to split a paragraph into word ranges and then traverses
 author: OfficeDev
 host: WORD
 api_set:
-    WordApi: 1.1
+    WordApi: 1.3
 script:
     content: |-
-        $("#setup").click(setup);
-        $("#highlight").click(highlightWords);
-
-        function highlightWords() {
-            Word.run(function (context) {
-                var myPars = context.document.body.paragraphs;
-                context.load(myPars);
-                return context.sync()
-                    .then(function () {
-                        var myWords = myPars.items[0].split([" "], true /*used to trim delimiters*/, true /* used to trim spaces */);
-                        context.load(myWords, { expand: 'font' });
-                        return context.sync()
-                            .then(function () {
-                                return forEach(myWords, function (item, i) {
-                                    if (i >= 1) {
-                                        myWords.items[i - 1].font.highlightColor = "#FFFFFF";
-                                    }
-                                    myWords.items[i].font.highlightColor = "#FFFF00";
-                                    return createTimerPromise(200).then(context.sync);
-                                })
-                            })
-                    })
-            })
-                .catch(OfficeHelpers.Utilities.log);
-
-            function createTimerPromise(ms) {
-                return new OfficeExtension.Promise(function (resolve) {
-                    setTimeout(resolve, ms);
-                })
-            }
+        $("#setup").click(() => tryCatch(setup));
+        $("#highlight").click(() => tryCatch(highlightWords));
 
-            function forEach(collection, handler) {
-                var promise = new OfficeExtension.Promise(function (resolve) { resolve(); });
-                collection.items.forEach(function (item, index) {
-                    promise = promise.then(function () {
-                        return handler(item, index);
-                    })
-                });
-                return promise;
-            }
+        async function highlightWords() {
+            await Word.run(async (context) => {
+                let paragraph = context.document.body.paragraphs.getFirst();
+                let words = paragraph.split(
+                    [" "], true /* trimDelimiters*/, true /* trimSpaces */);
+                words.load("text");
```

Comment:
Do we really need to load the text of the words here? The text property is never read later in the function.

Reply:
we don't needed.